lib/rpush/client/active_model/apns/notification.rb

```diff
@@ -7,15 +7,23 @@ module Notification
           APNS_PRIORITY_IMMEDIATE = 10
           APNS_PRIORITY_CONSERVE_POWER = 5
           APNS_PRIORITIES = [APNS_PRIORITY_IMMEDIATE, APNS_PRIORITY_CONSERVE_POWER]
+          MAX_PAYLOAD_BYTESIZE = 2048
```

Comment:
This is the correct maximum payload size that Apple enforces on APNS 1:

For other reviewers, here's the [relevant documentation](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/CreatingtheNotificationPayload.html#:~:text=The%20maximum%20size%20of%20the,size%20is%205KB%20(5120%20bytes)):

> If you are using the legacy APNs binary interface to send notifications instead of an HTTP/2 request, the maximum payload size is 2KB (2048 bytes)